en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 # Ballerina Expression Syntax
 
-Ballerina expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you with the common expressions that you can use while developing Choreo applications.
+Ballerina expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for method invocations, declaring a variable, etc. This document presents you with the common expressions that you can use while developing in the Choreo console.
 
-## Supported Literals 
+## Variables
+A variable declaration statement declares a value of a specific type (`string`, `integer`, etc.).  You can use a previously declared variable at a later phase in the logical process of your service or integration.
 
-### String
+### int
 
-A string is simply a sequence of characters. It is required to use double quotes (`"`)  to mark the boundary between the starting and ending characters.
+The `int` type is 64-bit signed integers (same as long in Java). A signed integer is an integer with the  `+` or `-` character in the front. However, You can also define a positive integer or zero using only numerical characters. The `int` type supports the usual [arithmetic operators: + - / %](#arithmetic-operators). The operator precedence is the same as C. Integer overflow is a runtime error in Ballerina. 
 
-#### Examples
+###### Examples:
 
-  ```ballerina
-  "Colombo"
-  ```
-  ```ballerina
-  "12345"
-  ```
-  ```ballerina
-  "Ballerina is a programming language intended for network-distributed applications."
-  ```
-
-### Signed Integer
-
-A signed integer is an integer with the  `+` or `-` character in the front, indicating the sign. You can also define a positive integer or zero using only numerical characters without the sign. Using the signed integer type, you can give positive integers, negative integers, and zero as an input to an expression-supported field.
-
-#### Examples
+To initialize a variable of type `int` in Choreo, you can use one of the following expressions:
 
   ```ballerina
   1234
-  ```
-  ```ballerina
+  
   -10
-  ```
-  ```ballerina
+  
   +2020
   ```
 
-### Decimal Floating-Point Number
+### float
+
+The `float` type is IEEE 64-bit binary floating-point (same as double in Java) and supports the same arithmetic operators as `int`. You have the option to use `+` or `-` characters in the front to indicate the sign.
```

Comment:
`double` should be formatted with black ticks.

Reply:
done.